master/static/srv/proto_get_trials_for_experiment.sql

```diff
@@ -0,0 +1,16 @@
+SELECT t.id AS id,
+    t.start_time AS start_time,
+    t.end_time AS end_time,
+    t.experiment_id AS experiment_id,
+    t.hparams AS hparams,
+    'STATE_' || t.state AS state,
+    (
+        SELECT s.prior_batches_processed + s.num_batches
+        FROM steps s
+        WHERE s.trial_id = t.id
+            AND s.state = 'COMPLETED'
+        ORDER BY s.id DESC
+        LIMIT 1
+    ) AS batches_processed
```

Comment:
I think I said this somewhere else but this should be `total_batches_processed` since this name is used elsewhere too.

Reply:
oh did I not update this? :man_facepalming:

Reply:
there it is https://github.com/determined-ai/determined/commit/af45a925e215328d1e2d0bb9633a5a5e13d004d0 wrong branch